javascript/javascript1/week2/exercises.js

```diff
@@ -0,0 +1,133 @@
+// STEP 2 @sergii-borodin
+
+// Step 3: Smart-ease - Goes Global!
+
+// Flight booking fullname function
+
+function getFullName(firstname, surname) {
+  const fullName = firstname + " " + surname;
+  return console.log(fullName);
+}
+
+const fullname1 = getFullName("Benjamin", "Hughes");
+const fullname2 = getFullName("Richard", "Chaplin");
+
+// Formal fullname
+
+function getFullname(firstname, surname, useFormalName, gender) {
+  const fullName = firstname + " " + surname;
+
+  if (useFormalName && gender === "male") {
+    const formalMaleName = "Lord " + fullName;
+    console.log(formalMaleName);
+    return formalMaleName;
+  } else if (useFormalName && gender === "female") {
+    const formalFemaleName = "Lady " + fullName;
+    console.log(formalFemaleName);
+    return formalFemaleName;
+  } else if (!useFormalName && gender === "male") {
+    const formalMaleName = "Lord " + fullName;
+    console.log(formalMaleName);
+    return formalMaleName;
+  } else if (!useFormalName && gender === "female") {
+    const formalFemaleName = "Lady " + fullName;
+    console.log(formalFemaleName);
+    return formalFemaleName;
+  } else if (
+    useFormalName === null ||
+    gender !== "male" ||
+    gender !== "female"
+  ) {
+    return console.log(
+      "To choose gender and a way, how you want to be addressed(formally or not) is required!"
+    );
+  }
+}
+
+getFullname("Benjamin", "Hughes", true, "male");
+getFullname("Barbara", "Hughes", true, "female");
+getFullname("Benjamin", "Hughes", false, "male");
+getFullname("Jenifer", "Hughes", false, "female");
+getFullname("Benjamin", "Hughes", "male");
+getFullname("Benjamin", "Hughes", true);
+
+// Event application
+
+function getEventWeekday(daysToEvent) {
+  const currentDate = new Date();
+  const currentDay = currentDate.getDay();
+  const numerDayOfWeek = (currentDay + daysToEvent) % 7;
+  const daysOfWeek = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+  const dayOfEventHeld = daysOfWeek[numerDayOfWeek];
+  console.log(currentDate);
+  console.log(
+    `A message from calendar: 'Remember that you have an event on ${dayOfEventHeld}'`
+  );
```

Comment:
Very good use of remainder operator. This is a very clean implementation of this method. You could look into using native ways of getting the day name (toLocaleDateString). But i lik the way you have done it.

Remember to return something, instead of just logging it